tests/kernel/timer/timer_behavior/Kconfig

```
@@ -31,3 +31,6 @@ config TIMER_TEST_PERIOD_MAX_DRIFT_PERCENT
 	default 10
 	help
 	  A value of 10 means 10%.
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 8192 if SOC_FAMILY_NRF
```

Shouldn't it be more specific, like this:
```suggestion
	default 8192 if NRF_RTC_TIMER && TICKLESS_KERNEL
```

---

Yes, otherwise it increases the tick rate from 128 to 8192 when not running tickless.

---

However, now I am checking both this PR and your suggestion and `SYS_CLOCK_TICKS_PER_SEC` tends to default to 32768. I have no idea why it seemed to work earlier (or was it just me operating not on pristine build tree earlier).

---

Place this before `source "Kconfig.zephyr"` so that this default is processed as first.